sql-plugin/src/main/scala/com/nvidia/spark/rapids/AlluxioUtils.scala

```diff
@@ -204,59 +201,69 @@ object AlluxioUtils extends Logging {
     (scheme + "://", bucket)
   }
 
-  private def runAlluxioCmd(param : String) : (Int,
-    scala.collection.mutable.ArrayBuffer[String]) = {
-    val params = alluxioCmd.tails.collect {
-        case Seq(first, _, _*) => first
-        case Seq(last) => last + " " + param
-      }.toSeq
-    val out : scala.collection.mutable.ArrayBuffer[String] =
-      new scala.collection.mutable.ArrayBuffer[String](10)
-    val ret = if (params.length == 1) {
-      // if alluxioCmd is like "alluxio fs mount", run by Process(String)
-      Process(params.last).!(ProcessLogger(out += _, _ => Unit))
-    } else {
-      // if alluxioCmd has multiple strings in the seq like "su ubuntu -c 'alluxio fs mount'",
-      // need to run by Process(Seq[String])
-      Process(params).!(ProcessLogger(out += _, _ => Unit))
+  private def getS3ClientConf(
+      alluxioUser: String,
+      s3AccessKey: Option[String],
+      s3SecretKey: Option[String]): InstancedConfiguration = {
+    val conf = InstancedConfiguration.defaults
+    s3AccessKey.foreach(access => conf.set(PropertyKey.S3A_ACCESS_KEY, access, Source.RUNTIME))
+    s3SecretKey.foreach(secret => conf.set(PropertyKey.S3A_SECRET_KEY, secret, Source.RUNTIME))
+    alluxioMasterHost.foreach(host =>
+      conf.set(PropertyKey.MASTER_HOSTNAME, host, Source.RUNTIME))
+    alluxioMasterPort.foreach(port =>
+      conf.set(PropertyKey.MASTER_RPC_PORT, port, Source.RUNTIME))
+    conf.set(PropertyKey.SECURITY_LOGIN_USERNAME, alluxioUser, Source.RUNTIME)
+    conf
+  }
+
+  private def getExistS3MountPoints(
+      alluxioUser: String,
+      s3AccessKey: Option[String],
+      s3SecretKey: Option[String]): mutable.Map[String, MountPointInfo] = {
+    val conf = getS3ClientConf(alluxioUser, s3AccessKey, s3SecretKey)
+    // get s3 mount points by alluxio client
+    withResource(alluxio.client.file.FileSystem.Factory.create(conf)) { fs =>
+      val mountTable = fs.getMountTable
+      mountTable.asScala.filter(e => e._2.getUfsType == "s3")
```

Comment:
ufs type will always be s3 and not s3a or something like that?

this woudl be more readable like:
`      mountTable.asScala.filter{ case (_, mountPoint) => mountPoint.getUfsType == "s3"}
`

Reply:
Checked it should be "s3", anyway also added "s3a"

[S3AUnderFileSystem](https://github.com/Alluxio/alluxio/blob/v2.8.0/underfs/s3a/src/main/java/alluxio/underfs/s3a/S3AUnderFileSystem.java#L341-L344)
  
public String getUnderFSType() {
    return "s3";
  }